components/GlobalButton/index.js

```diff
@@ -0,0 +1,9 @@
+import * as Styled from "./GlobalButton.styled";
+
+export default function GlobalButton({ name, onClick, type, textColor }) {
+  return (
+    <Styled.globalButton onClick={onClick} type={type} textColor={textColor}>
+      {name}
+    </Styled.globalButton>
+  );
+}
```

Comment:
Is there a specific reason why you wrap the Button component in a second "GlobalButton Component? You can use the Styled.globalButton component directly without the need for this extra component.
Simply import the GlobalButton in the other places and use them instead of this component.